src/window-service.js

```diff
@@ -177,6 +178,27 @@
             mainWindow.addEventListener('closed', closedEvent);
         }
 
+        snapToScreenBounds(targetWindow) {
+            fin.desktop.System.getMonitorInfo((info) => {
+                targetWindow.getBounds((bounds) => {
+
+                    if (bounds.left < info.virtualScreen.left) {
+                        bounds.left = info.virtualScreen.left;
+                    } else if (bounds.left + bounds.width > info.virtualScreen.right) {
+                        bounds.left = info.virtualScreen.right - bounds.width;
+                    }
+
+                    if (bounds.top < info.virtualScreen.top) {
+                        bounds.top = info.virtualScreen.top;
```

Comment:
It does, http://cdn.openfin.co/jsdocs/stable/tutorial-system.getMonitorInfo.html, should I add in taskbar compensation too?